docs/config.toml

```diff
@@ -79,3 +79,19 @@ copyright = "The Kurator Authors"
 	url = "https://groups.google.com/g/kurator-dev"
 	icon = "fa fa-envelope"
   desc = "Kyverno mailing list"
```

Comment:
? this is wrong

Reply:
"Kyverno mailing list"?

we can fix this spelling mistake

Reply:
Kurator mailing list

Reply:
yes, fix it in https://github.com/kurator-dev/kurator/pull/455